addons/website/static/src/snippets/s_process_steps/002.scss

```diff
@@ -34,12 +46,6 @@
         }
     }
 
-    &.s_process_steps_connector_curved_arrow {
-        .s_process_step:nth-child(odd) .s_process_step_connector {
-            transform: scale(1, -1);
-        }
-    }
-
```

Comment:
Why did we have both this and the related JS code in the snippet option? Do you know? Can you really remove it? Won't you break existing snippets in user databases?